packages/postgres-database/src/simcore_postgres_database/migration/versions/af5de00bf4cf_.py

```diff
@@ -0,0 +1,53 @@
+"""empty message
+
+Revision ID: af5de00bf4cf
+Revises: 71ea254837b0
+Create Date: 2023-06-19 08:41:09.835411+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "af5de00bf4cf"
+down_revision = "71ea254837b0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "projects_comments",
+        sa.Column("comment_id", sa.BigInteger(), autoincrement=True, nullable=False),
+        sa.Column("project_uuid", sa.String(), nullable=False),
+        sa.Column("user_id", sa.BigInteger(), nullable=False),
+        sa.Column("content", sa.String(), nullable=True),
```

Comment:
I think it should be "contents"
https://proofed.com/writing-tips/word-choice-content-vs-contents/#:~:text=Content%20is%20an%20uncountable%20noun,%E2%80%9Ctable%20of%20contents%E2%80%9D).

Maybe check with an english speaker...